src/controllers/JobRoleController.ts

```diff
@@ -8,4 +8,13 @@ export const getAllJobRoles = async (req: express.Request, res: express.Response
         res.locals.errormessage = e.message;
         res.render('openJobRoleList.html');
     }
+}
+
+export const getJobRoleById = async (req: express.Request, res: express.Response): Promise<void> => {
+    try {
+        res.render('openJobRoleDetail.html', { openJobRole: await getSingleJobRole(req.params.id) });
```

Comment:
Don't change it for here but avoid being specific when you don't need to be. The file could be called jobRoleDetail as it can be used for roles of any status.